player/lua/stats.lua

```diff
@@ -866,8 +866,8 @@ local function add_video_out(s)
     append_display_sync(s)
     append_perfdata(s, false, o.print_perfdata_passes)
 
-    if mp.get_property_native("deinterlace") then
-        append_property(s, "deinterlace", {prefix="Deinterlacing:"})
+    if mp.get_property_native("deinterlace-active") then
+        append_property(s, "deinterlace-active", {prefix="Deinterlacing:"})
```

Comment:
this could be "deinterlace", so it would show "yes" or "auto" when active.

Reply:
But on "auto" you wouldn't know if it's currently on or not.

Reply:
It is hidden if not enabled.

Reply:
Oh you mean `deinterlace-active` for the check and `deinterlace` for the text, yes that would work.

Reply:
Good idea.